libp2p/transports/tcptransport.nim

```diff
@@ -190,6 +190,7 @@ method stop*(self: TcpTransport) {.async.} =
     await allFutures(toWait)
 
     self.servers = @[]
+    self.acceptFuts = @[]
```

Comment:
I don't know if this is the right approach as I don't have much knowledge about this code. @dryajov could you please advise?

Reply:
We need to make sure that all the accept futs are properly canceled, before resetting.